android/app/src/main/java/org/fossasia/openevent/adapters/viewholders/DayScheduleViewHolder.java

```diff
@@ -123,10 +123,9 @@ public void bindSession(RealmDataRepository realmRepo) {
                             });
 
                     realmRepo.setBookmark(sessionId, true).subscribe();
+
                     slotBookmark.setImageResource(R.drawable.ic_bookmark_white_24dp);
                     slotBookmark.setColorFilter(storedColor, PorterDuff.Mode.SRC_ATOP);
-                    if (onBookmarkSelectedListener != null)
-                        onBookmarkSelectedListener.showSnackbar(new BookmarkStatus(storedColor, sessionId, CODE_UNDO_ADDED));
```

Comment:
Are you sure this line needs to be deleted?

Reply:
Yes , I have checked it locally in my phone.

Reply:
@iamareebjamal I have updated the branch. Please review it again and merge it. If there are some other changes to be done, tell me.

Reply:
I am AFK. I'll do it ASAP